src/ebfloeseg/load.py

```diff
@@ -99,7 +112,7 @@ def load(
     payload = {
         "REQUEST": "GetSnapshot",
         "TIME": datetime,
-        "BBOX": bbox,
+        "BBOX": ",".join(str(f) for f in bbox),
```

Comment:
What do you say about encapsulating this functionality in a class property of `BoundingBox`? It could based on the `__repr__` or the `__str__` methods.

Reply:
My concern here is the same as above – I don't think this is fundamentally a thing the `BoundingBox` should be able to do. In this case, the string generation is very specific to making this payload, and other payloads might have slightly different requirements. I've made this line a bit more explicit.